recipes/patchelf/all/conanfile.py

```diff
@@ -23,11 +28,16 @@ def build_requirements(self):
         self.build_requires("libtool/2.4.6")
 
     def validate(self):
-        if not tools.is_apple_os(self.settings.os) and self.settings.os not in ("FreeBSD", "Linux"):
+        if not is_apple_os(self) and self.settings.os not in ("FreeBSD", "Linux"):
```

Comment:
```suggestion
        if not is_apple_os(self) and self.info.settings.os not in ("FreeBSD", "Linux"):
```
On validate() method, use self.info.options/settings. It's a migration to Conan 2.0